### PR TITLE
Java 8 migration. Remove WebSocketUtil.randomNumber

### PR DESCRIPTION
**Pull request by doom369**
Motivation:

Use `ThreadLocalRandom.current().nextInt()` directly instead of `(int) ThreadLocalRandom.current().nextDouble()`

### CODE REVIEW

**Comment by doom369** [comment]
@normanmaurer found 2 more places that could be improved. Pushed.

File: codec-http/src/main/java/io/netty/handler/codec/http/websocketx/WebSocketClientHandshaker00.java [hunk 1]
```diff
@@ -201,20 +202,20 @@ protected void verify(FullHttpResponse response) {
...
     private static String insertRandomCharacters(String key) {
-        int count = WebSocketUtil.randomNumber(1, 12);
+        int count = ThreadLocalRandom.current().nextInt(1, 13);
 
         char[] randomChars = new char[count];
         int randCount = 0;
         while (randCount < count) {
-            int rand = (int) (Math.random() * 0x7e + 0x21);
+            int rand = ThreadLocalRandom.current().nextInt(0x21, 0x7e + 0x21);
             if (0x21 < rand && rand < 0x2f || 0x3a < rand && rand < 0x7e) {
```
Comment (doom369) [inline codec-http/src/main/java/io/netty/handler/codec/http/websocketx/WebSocketClientHandshaker00.java]: @normanmaurer this check also doesn't seem correct to me.
Random is generated in range `0x21 .. 0x7e + 0x21`, however, values above` 0x7e` are skipped...

File: codec-http/src/main/java/io/netty/handler/codec/http/websocketx/WebSocketClientHandshaker00.java [hunk 2]
```diff
@@ -201,20 +202,20 @@ protected void verify(FullHttpResponse response) {
...
+        int count = ThreadLocalRandom.current().nextInt(1, 13);
 
         char[] randomChars = new char[count];
         int randCount = 0;
         while (randCount < count) {
-            int rand = (int) (Math.random() * 0x7e + 0x21);
-            if (0x21 < rand && rand < 0x2f || 0x3a < rand && rand < 0x7e) {
+            int rand = ThreadLocalRandom.current().nextInt(0x22, 0x7e);
+            if (rand < 0x2f || 0x3a < rand) {
                 randomChars[randCount] = (char) rand;
```
Comment (doom369) [inline codec-http/src/main/java/io/netty/handler/codec/http/websocketx/WebSocketClientHandshaker00.java]: Simplified above check, as `nextInt` generates `[0x22, 0.7e)`, so rand is always `> 0x21` and always `< 0x7e`

**Comment by normanmaurer** [comment]
@netty-bot test this please

**Comment by normanmaurer** [comment]
@doom369 thanks!

**Pull request merged**